NanoProd/crab/Run3_2022EE/DY.yaml

```diff
@@ -3,7 +3,7 @@ config:
     sampleType: mc
     era: Run3_2022EE
 
-DYJetsToLL_M-50: /DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v5-v2/MINIAODSIM
+DYJetsToLL_M-50-madgraphMLM: /DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v5-v2/MINIAODSIM
 DYJetsToLL_M-50_forPOG: /DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-forPOG_130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 DYTo2L_MLL-4to50_Poisson60: /DYToLL_M-4To50_TuneCP5_13p6TeV-pythia8/Run3Summer22EEMiniAODv4-Poisson60KeepRAW_130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 DYTo2L_MLL-50_Poisson60: /DYToLL_M-50_TuneCP5_13p6TeV-pythia8/Run3Summer22EEMiniAODv4-Poisson60KeepRAW_130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
```

Comment:
use new notation `DYto2L`

Reply:
Updated to new notation and sorted out exceptions as well.